pkg/command/plugin.go

```diff
@@ -38,7 +39,9 @@ var (
 )
 
 const (
-	invalidTargetMsg = "invalid target specified. Please specify a correct value for the `--target/-t` flag from '" + common.TargetList + "'"
+	invalidTargetMsg               = "invalid target specified. Please specify a correct value for the `--target/-t` flag from '" + common.TargetList + "'"
+	errorWhileDiscoveringPlugins   = "there was an error while discovering plugins, error information: '%v'"
+	errorWhileGettingServerPlugins = "there was an error while getting installed server plugins, error information: '%v'"
```

Comment:
Dependent on if we remove the warning printouts or not (see further comment):
Since this error message is user-facing, let's use the word "context" instead of "server":
```suggestion
	errorWhileGettingContextPlugins = "there was an error while getting installed context plugins, error information: '%v'"
```